### PR TITLE
Fixes/#466 e hv clustering not working after merging dijkstra clustering

### PR DESCRIPTION
**Pull request by CarlosEpia**
Solves #466.

### CODE REVIEW

File: etrago/cluster/networkclustering.py [hunk 1]
```diff
@@ -781,23 +781,81 @@ def fetch():
...
+                    
+                    ----------------------- WARNING ---------------------------
+                    THE FOLLOWING BUSES WERE DELETED BECAUSE THEY WERE ISOLATED:
+                        {delete_buses.to_list()}.
+                    IT IS POTENTIALLY A SIGN OF A PROBLEM IN THE DATASET
+                    ----------------------- WARNING ---------------------------
+                    
+                    """)
+                    
+    network.buses = network.buses.drop(delete_buses)
```
Comment (ClaraBuettner) [inline etrago/cluster/networkclustering.py:826]: These lines are working, but in general I would recommend to use the build-in pypsa functions to drop components (see e.g. https://github.com/openego/eTraGo/blob/ad10466425386b2aa42a6f78c4f03d562188783d/etrago/tools/utilities.py#L1712-L1715). When you use them, e.g. time series are dropped automatically.

File: etrago/cluster/networkclustering.py [hunk 2]
```diff
@@ -781,23 +781,81 @@ def fetch():
...
         busmap = busmap_from_psql(self)
 
         self.network, busmap = cluster_on_extra_high_voltage(
-            self.network, busmap, with_time=True
+            self, busmap, with_time=True
         )
 
         self.update_busmap(busmap)
-
+        buses_by_country(self)
```
Comment (ClaraBuettner) [inline etrago/cluster/networkclustering.py:858]: I think this should be `self.buses_by_country()` , right?

File: etrago/cluster/networkclustering.py [hunk 3]
```diff
@@ -1409,9 +1471,13 @@ def kmedoids_dijkstra_clustering(etrago):
...
 
     if self.args["network_clustering"]["active"]:
+        
+        if self.args["network_clustering_ehv"]:    
+        
+            self.adapt_crossborder_buses()
 
         self.network.generators.control = "PV"
-
+
```
Comment (ClaraBuettner) [inline etrago/cluster/networkclustering.py:1480]: It looks like you should run `black` again...

**Comment by ClaraBuettner** [comment]
Unfortunately I always run into errors when I try to use the kmedoids-dijkstra-clustering. I was only able to test eHV clustering, which was working as expected now. 
I would like to be able to test both methods before adding my review.

File: etrago/appl.py [hunk 4]
```diff
@@ -119,7 +119,7 @@
             "rural_heat": {"base": ["CH4", "AC"], "strategy": "consecutive"},
         },
     },
-    "network_clustering_ehv": False,  # clustering of HV buses to EHV buses.
+    "network_clustering_ehv": True,  # clustering of HV buses to EHV buses.
```
[the remark this answers is not in the document]
Comment (CarlosEpia) [reply etrago/appl.py]: network_clustering_ehv set to False again.

File: etrago/cluster/networkclustering.py (hunk 1 above)
Comment (CarlosEpia) [reply etrago/cluster/networkclustering.py]: The function mremove is used now to delete elements.

File: etrago/cluster/networkclustering.py (hunk 2 above)
Comment (CarlosEpia) [reply etrago/cluster/networkclustering.py]: buses_by_country() was aggregated to the eTraGo methods.

File: etrago/cluster/networkclustering.py (hunk 3 above)
Comment (CarlosEpia) [reply etrago/cluster/networkclustering.py:1481]: Black was used again.

**Comment by CarlosEpia** [comment]
@ClaraBuettner I think this branch is ready for a second check. Since I merged dev into it, I expect that it works even with the dataset from the 16th of August.

**Pull request merged**